README.md

```diff
@@ -40,3 +40,5 @@ When you're done working on the site, run the following command to move the post
 ```bash
 bundle exec rake integrate
 ```
+##
```

Comment:
Why the `##`?

Reply:
Just to add a separator but we can leave out if you'd prefer

Reply:
Removed